web/bootstrap/.gitignore

```diff
@@ -1,26 +0,0 @@
-.DS_Store
```

Comment:
whats all the stuff in this directory`web/boostrap` for? Did the bootstrap node have a UI? I'm confused

Reply:
The kiev team made a web interface for the bootstrap node but we never ended up using it. So we're removing it and maybe one day we could repurpose some of the code

Reply:
ah interesting